jrnl/install.py

```diff
@@ -96,20 +96,23 @@ def load_or_install_jrnl():
         log.debug("Reading configuration from file %s", config_path)
         config = util.load_config(config_path)
 
-        try:
-            upgrade.upgrade_jrnl_if_necessary(config_path)
-        except upgrade.UpgradeValidationException:
-            print("Aborting upgrade.", file=sys.stderr)
-            print(
-                "Please tell us about this problem at the following URL:",
-                file=sys.stderr,
-            )
-            print(
-                "https://github.com/jrnl-org/jrnl/issues/new?title=UpgradeValidationException",
-                file=sys.stderr,
-            )
-            print("Exiting.", file=sys.stderr)
-            sys.exit(1)
+        if util.is_old_version(config_path):
```

Comment:
I agree that this is good to break out into a separate function, but without also updating `upgrade_jrnl_if_necessary` we now perform this check twice.